zcash_client_sqlite/src/testing.rs

```diff
@@ -1099,24 +1152,22 @@ fn fake_compact_block_spending<P: consensus::Parameters, Fvk: TestFvk>(
     let mut rng = OsRng;
     let mut ctx = fake_compact_tx(&mut rng);
 
-    // Create a fake spend
-    fvk.add_compact_spend(&mut ctx, nf, &mut rng);
-
-    // Create a fake Note for the payment
-    ctx.outputs
-        .push(compact_sapling_output(params, height, to, value, fvk.sapling_ovk(), &mut rng).0);
-
-    // Create a fake Note for the change
-    fvk.add_compact_output(
+    // Create a fake spend and a fake Note for the change
+    fvk.add_compact_action(
         &mut ctx,
         params,
         height,
+        nf,
         AddressType::Internal,
         (in_value - value).unwrap(),
         initial_sapling_tree_size,
```

Comment:
Why does this take the Sapling tree size, and not the Orchard tree size?

Reply:
Sailing nullifiers depend on tree position, Orchard nullifiers don't.